packages/remirror__extension-react-tables/src/views/table-view.tsx

```diff
@@ -0,0 +1,259 @@
+import { css } from '@emotion/css';
+import { h } from 'jsx-dom';
+import { EditorSchema, EditorView, NodeView, range, throttle, Transaction } from '@remirror/core';
+import { Node as ProsemirrorNode } from '@remirror/pm/model';
+import { TableMap, updateColumnsOnResize } from '@remirror/pm/tables';
+import { Decoration } from '@remirror/pm/view';
+import { ExtensionTablesTheme } from '@remirror/theme';
+
+import TableInsertButton, { shouldHideInsertButton } from '../components/table-insert-button';
+import { ReactTableNodeAttrs } from '../table-extensions';
+import { injectControllers } from '../utils/controller';
+import { setNodeAttrs } from '../utils/prosemirror';
+
+export type GetTableStyle = (attrs: ReactTableNodeAttrs) => { className: string; style: string };
+
+export function buildTableStyle(): GetTableStyle {
+  const previewSelectionBorderColor = '#0067ce';
```

Comment:
Instead of hardcoding the color here: How about using one of the [existing theme colors](https://github.com/remirror/remirror/blob/beta/packages/remirror__styles/theme.css#L18) (or add a new color)?
This eases styling the tables in line with the rest of the editor.